StabilityPackage.wl

```diff
@@ -63,13 +82,57 @@ SelectSingleRadius[data_,ri_]:=Association[
 "mids"->data["mids"] (*Cos\[Theta] bin midpoints*)
 ]
 
+Options[siPotential]={"xflavor"-> False};
```

Comment:
lots of this looks like it is duplicated below. Is there no way to avoid the duplication?